src/observe.ts

```diff
@@ -0,0 +1,330 @@
+import has from './has';
+import { Handle, Hash } from './interfaces';
+import { getPropertyDescriptor, isIdentical } from './lang';
```

Comment:
`ObjectObserver.ts` used to have its own version of `getPropertyDescriptor` which also included default behavior if no descriptor yet existed.  The version in `lang` doesn't have that behavior.  Is this change intentional or should this be preserving the behavior of `getPropertyDescriptor` as it existed in `ObjectObserver`?

(Incidentally, #31 was going to remove `getPropertyDescriptor` from lang anyway, so I'd probably opt to keep the internal function in this module.)